codecs/imagequant/imagequant.cpp

```diff
@@ -61,21 +61,21 @@ RawImage quantize(std::string rawimage, int image_width, int image_height, int n
 }
 
 const liq_color zx_colors[] = {
-  {.a = 255, .r = 0, .g = 0, .b = 0},       // regular black
-  {.a = 255, .r = 0, .g = 0, .b = 215},     // regular blue
-  {.a = 255, .r = 215, .g = 0, .b = 0},     // regular red
-  {.a = 255, .r = 215, .g = 0, .b = 215},   // regular magenta
-  {.a = 255, .r = 0, .g = 215, .b = 0},     // regular green
-  {.a = 255, .r = 0, .g = 215, .b = 215},   // regular cyan
-  {.a = 255, .r = 215, .g = 215, .b = 0},   // regular yellow
-  {.a = 255, .r = 215, .g = 215, .b = 215}, // regular white
-  {.a = 255, .r = 0, .g = 0, .b = 255},     // bright blue
-  {.a = 255, .r = 255, .g = 0, .b = 0},     // bright red
-  {.a = 255, .r = 255, .g = 0, .b = 255},   // bright magenta
-  {.a = 255, .r = 0, .g = 255, .b = 0},     // bright green
-  {.a = 255, .r = 0, .g = 255, .b = 255},   // bright cyan
-  {.a = 255, .r = 255, .g = 255, .b = 0},   // bright yellow
-  {.a = 255, .r = 255, .g = 255, .b = 255}  // bright white
+  {.r = 0, .g = 0, .b = 0, .a = 255},       // regular black
```

Comment:
Note: I made this change to accomodate for a new C++ warning about strict order of initialization.